runtime/integration-tests/src/generic/utils/democracy.rs

```diff
@@ -1,6 +1,6 @@
 use std::ops::Add;
 
-use cfg_primitives::{Balance, BlockNumber, CouncilCollective};
```

Comment:
With this PR, do we no longer need the old democracy? If not, I think this file can be fully removed. Just the gateway tests are using them, but they can done though Root instead.